issuance/issuance.go

```diff
@@ -203,6 +191,16 @@ func NewProfile(profileConfig ProfileConfig, issuerConfig IssuerConfig) (*Profil
 	if issuerConfig.OCSPURL == "" {
 		return nil, errors.New("OCSP URL is required")
 	}
+
+	var policies []asn1.ObjectIdentifier
+	for _, policyConfig := range profileConfig.Policies {
+		oid, err := parseOID(policyConfig.OID)
+		if err != nil {
+			return nil, fmt.Errorf("failed parsing policy OID %q: %w", policyConfig.OID, err)
```

Comment:
Optional: follow the style of some of our newer errors

```suggestion
			return nil, fmt.Errorf("parsing policy OID %q: %w", policyConfig.OID, err)
```

Reply:
Gonna leave as-is to submit this now so that I can unblock https://github.com/letsencrypt/boulder/pull/6992